### PR TITLE
add PTB version check

### PR DESCRIPTION
**Pull request by Remi-Gau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,25 @@
...
+For instructions see the following links:
+
+| Requirements                                             | Used version |
+|----------------------------------------------------------|--------------|
+| [PsychToolBox](http://psychtoolbox.org/)  Duuuuhh        | >3.0.13      |
+| [Matlab](https://www.mathworks.com/products/matlab.html) | 201??        |
+| or [octave](https://www.gnu.org/software/octave/)        | 4.?          |
+
+The exact version required for this to work but it is known to work with:
+-   matlab 2017a or octave 4.2.2 and PTB 3.0.16.
```
Comment (Remi-Gau) [inline README.md:24]: @marcobarilari maybe you can add a line here that says which versions you have tried

Comment (marcobarilari) [reply README.md:24]: ok, I have the 2017b and same ptb version. Do you think adding this version would be very informative?

Comment (marcobarilari) [reply README.md:24]: ok just sent a pr with the info added, but feel free to ignore/edit it

**Comment by marcobarilari** [comment]
it is good to go for me :)

**Pull request merged**